torchgeo/trainers/chesapeake.py

```diff
@@ -31,8 +32,20 @@
 DataLoader.__module__ = "torch.utils.data"
 Module.__module__ = "torch.nn"
 
-CMAP = matplotlib.colors.ListedColormap(
-    [np.array(Chesapeake7.cmap[i + 1]) / 255.0 for i in range(6)]
+CMAP_7 = matplotlib.colors.ListedColormap(
```

Comment:
Can you add a TODO saying that this stuff needs to be moved to the `ChesapeakeCVPR` class in the future?